backend/nlp/src/new/extractor.py

```diff
@@ -34,13 +35,17 @@ def extract(raw_txt):
         summary_length = int(0.2 * len(cleaned_text_list))
         top_ranked = nlargest(summary_length, pageranks, key=pageranks.get)
         top_ranked.sort()
-        # print(result)
+        cl = Cluster()
+        top_ranked = cl.splitIntoParagraph(top_ranked, 25)
+        print(top_ranked)
 
         result = ''
-        for key in top_ranked:
-            top_ranked_sentence = m.sentences[key]
-            # print('.{}.'.format(top_ranked_sentence))
-            result += '{}. '.format(top_ranked_sentence)
+        for paragraph in top_ranked:
+            for key in paragraph:
+                top_ranked_sentence = m.sentences[key]
+                # print('.{}.'.format(top_ranked_sentence))
+                result += '{}. '.format(top_ranked_sentence)
+            result += '<br /><br />\n'
```

Comment:
Would this work (output correctly) if you used `\n` instead of `<br/>`? The summary stuff shouldn't know about HTML and it would mean we don't have to do the `dangerouslySetInnerHTML` in the frontend file.

Reply:
i tried `result += '<br /><br />\n` and it is not working